client/db_test.go

```diff
@@ -288,3 +289,31 @@ func TestDebugName(t *testing.T) {
 		return nil
 	})
 }
+
+func TestCommonMethods(t *testing.T) {
+	methods := []string{
```

Comment:
Ok, switched this around to using a blacklist instead. Not a very efficient implementation, but that doesn't matter.